server/src/routes/drugs/index.js

```diff
@@ -0,0 +1,14 @@
+import { Router } from 'express';
+import { StatusCodes } from 'http-status-codes';
+import Drug from '../../models/drug';
+
+const drugRouter = Router();
+
+drugRouter.get('/', async (req, res) => {
+  const drugs = await Drug.find({});
+  if (!drugs)
```

Comment:
Jaki rodzaj błędu chcemy tu przechwycić? Jeżeli połączenie z bazą wysiądzie to moongose rzuci błędem, jeżeli nie znajdzie nic to będzie pusta tablica (co jest ok). Wydaje mi się to sprawdzenie zbędne ale może jest coś o czym nie wiem 🤔 😉

Reply:
masz rację, poprawione